### PR TITLE
fix: (sfui2-1258) added missing accessibility notes for some of components

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue

https://vsf.atlassian.net/browse/SFUI2-1258?atlOrigin=eyJpIjoiOGMyZTE2NGM3NjY5NGQxZTlkMjc1NTkwNWFkNDNhOTgiLCJwIjoiaiJ9

# Scope of work

- updated accessibility notes

# Screenshots of visual changes

-

# Checklist

- [ ] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

File: apps/docs/components/components/badge.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ A common use case for the Badge is to place it on a user's avatar.
 <!-- end react -->
 </Showcase>
 
+## Accessibility Notes
+
+The component itseld does not provide any specific accessibility features. Please make sure that the content displayed in SfBadge component is meaningful and understable by all users.
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/badge.md:66]:
```suggestion
The component itself does not provide any specific accessibility features. Please make sure that the content displayed in SfBadge component is meaningful and understandable by all users.
```

File: apps/docs/components/components/counter.md [hunk 2]
```diff
@@ -68,34 +81,45 @@ The root element of `SfCounter` is a `<span>`. Since a `<span>` is an inline ele
...
 ::: slot api
+
 ## Props
 
-| Prop name | Type              | Default value | Possible values                       |
-|-----------|-------------------|---------------|---------------------------------------|
-| `size`      | `SfCounterSize` | `'base'`        | `'3xs'`, `'2xs'`, `'xs'`, `'sm'`, `'base'`, `'lg'` |
-| `pill`      | `boolean`         | `false`       |                                       |
+| Prop name | Type            | Default value | Possible values                                    |
+| --------- | --------------- | ------------- | -------------------------------------------------- |
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/counter.md:88]: question: wondering, are you doing this formatting of table manually or via some extension ?

File: apps/docs/components/components/drawer.md [hunk 3]
```diff
@@ -35,13 +39,21 @@ You can wrap `SfDrawer` in a <!-- vue -->[`<transition>`](https://vuejs.org/guid
...
 <!-- react -->
+
 <<<../../preview/next/pages/showcases/Drawer/TransitionAndCloseButton.tsx#source
+
 <!-- end react -->
 </Showcase>
 
+## Accessibility notes
+
+The component is providiing keyboard accessibility with `@keydown.esc="onEscKeyDown"` and `onClickOutside` function which allows users to close the drawer by pressing the Escape key or click outside of the drawer.
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/drawer.md:55]:
```suggestion
The component is providing keyboard accessibility with `@keydown.esc="onEscKeyDown"` and `onClickOutside` function which allows users to close the drawer by pressing the Escape key or click outside of the drawer.
```

File: apps/docs/components/components/link.md [hunk 4]
```diff
@@ -46,6 +55,11 @@ If you need more customization beyond the built-in variants, we recommend copyin
 :::
 
 <!-- react -->
+
+## Accessibility notes
+
+Link component can be rendered as an `<a>` or <!-- vue -->`<NuxtLink>`<!-- end vue --><!-- react -->`<NextLink>`<!-- end react --> or any other tag by providing it with prop <!-- vue -->`tag`<!-- end vue --><!-- react -->`as`<!-- end react -->. When no tag provided, the component will render as an `<a>`. To achieve proper accessibility it is important to implement required properies depending on the passed tag.
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/link.md:61]:
```suggestion
Link component can be rendered as an `<a>` or <!-- vue -->`<NuxtLink>`<!-- end vue --><!-- react -->`<NextLink>`<!-- end react --> or any other tag by providing it with prop <!-- vue -->`tag`<!-- end vue --><!-- react -->`as`<!-- end react -->. When no tag provided, the component will render as an `<a>`. To achieve proper accessibility it is important to implement required properties depending on the passed tag.
```

File: apps/docs/components/components/listitem.md [hunk 5]
```diff
@@ -47,13 +56,20 @@ For items with a lot of text content, you can use Tailwind's `truncate` class.
...
 <!-- react -->
+
 <<<../../preview/next/pages/showcases/ListItem/ListItemTruncated.tsx#source
+
 <!-- end react -->
 </Showcase>
 
+## Accessibility notes
+
+ListItem component can be rendered as an `<li>` or `<a>` or any other tag by providing it with prop <!-- vue -->`tag`<!-- end vue --><!-- react -->`as`<!-- end react -->. When no tag provided, the component will render as an `<li>`. To achieve proper accessibility it is important to implement required properies depending on the passed tag.
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/listitem.md:72]:
```suggestion
ListItem component can be rendered as an `<li>` or `<a>` or any other tag by providing it with prop <!-- vue -->`tag`<!-- end vue --><!-- react -->`as`<!-- end react -->. When no tag provided, the component will render as an `<li>`. To achieve proper accessibility it is important to implement required properties depending on the passed tag.
```

File: apps/docs/components/components/scrollable.md [hunk 6]
```diff
@@ -76,64 +99,77 @@ By default `SfScrollable` scroll by one page of items, but can be modified that
...
-| `isActiveIndexCentered` | `boolean`  |  |       |
-| `buttonPrevAriaLabel` | `string`  | `'Previous'` |       |
-| `buttonNextAriaLabel` | `string`  | `'Next'` |       |
+| Prop name               | Type                                               | Default value  | Possible values                   |
+| ----------------------- | -------------------------------------------------- | -------------- | --------------------------------- | --- |
+| `direction`             | `SfScrollableDirection`                            | `'horizontal'` | `'horizontal'`, `'vertical'`      |
+| `buttonsPlacement`      | `SfScrollableButtonsPlacement`                     | `'block'`      | `'block'`, `'floating'`, `'none'` |
+| `activeIndex`           | `number`                                           |                |                                   |
+| `reduceMotion`          | `boolean`                                          |                |                                   |
+| `drag`                  | `{ sensitivity: number; containerWidth: boolean; } | boolean`       |                                   |     |
```
Comment (Szymon-dziewonski) [inline apps/docs/components/components/scrollable.md:108]: Im not sure about this line, 
```
`{ sensitivity: number; containerWidth: boolean; } | boolean`
```
is `Type` which is fine but then we have extra column at line 103 which might cause render issue, please can you double check how it looks in docs ? :)

File: apps/docs/components/components/counter.md (hunk 2 above)
Comment (aniamusial) [reply apps/docs/components/components/counter.md:88]: extension, prettier maybe?

File: apps/docs/components/components/scrollable.md (hunk 6 above)
Comment (aniamusial) [reply apps/docs/components/components/scrollable.md]: @Szymon-dziewonski indeed these auto-linting caused the issue, I removed those changes and disabled auto-fixing on save, shoulld be all good now :)

**Review by Szymon-dziewonski — APPROVED** [review]

**Pull request merged**